### DISCUSSION TITLE
HyperHDR support the Sengled bulbs?

### DISCUSSION THREAD

**Discussion by appleimperio** [awawa-dev/HyperHDR#553] [Q&A]
Buy a few of these bulbs want to know if its posible to integrate them with HyperHDR. https://www.amazon.com/Sengled-Changing-Dimmable-Multicolor-Equivalent/dp/B0899LJHVV/ref=sr_1_2_sspa?crid=2XAFJYB41ICEU&keywords=sengled%2Bsmart%2Bbulb%2Bzigbee&qid=1681666025&sprefix=sengled%2Bsmart%2Bbulb%2Caps%2C70&sr=8-2-spons&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyMDBWODhDSkRHN1ZJJmVuY3J5cHRlZElkPUEwOTUzOTc1VDFBOTZaNzU1Mzg1JmVuY3J5cHRlZEFkSWQ9QTAxMTgwMTYzNVkxSjJPNkM2SUxFJndpZGdldE5hbWU9c3BfYXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ&th=1 They are zigbee and can use MQTT. Thanks

**Comment by awawa-dev** [comment]
Hi
If it doesn't support color streaming over UDP, it's pretty much useless to use with HyperHDR for ambient lighting at least: too much latency, too little refresh rate. This is where the giant advantage of Philips Hue comes into play: open, efficient and well-documented API.
